package.json

```diff
@@ -76,6 +76,7 @@
     "esbuild": "^0.18.10",
     "escape-string-regexp": "^5.0.0",
     "etag": "^1.8.1",
+    "firebase-functions": "^4.3.1",
```

Comment:
move to dev dependency

Reply:
@Hebilicious I agree it should be a dev dep, but I am getting a nitro build error when it is a dev dependancy. The build runs fine when it's a normal dependancy. I'm not sure where to start with that one?

The error I get when it's a dev dependancy is:

```
nitro ⚡ pnpm build                                                                                                              17:34:54

> nitropack@2.5.2 build /Users/JCGeek/Documents/Projects/nitro
> unbuild

ℹ Building nitropack                                                                                                        5:35:05 PM
Error building /Users/JCGeek/Documents/Projects/nitro: SyntaxError: Unexpected token (13:7) in /Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/firebase-functions@4.4.1_firebase-admin@11.9.0/node_modules/firebase-functions/lib/v2/providers/https.d.ts
SyntaxError: Unexpected token (13:7) in /Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/firebase-functions@4.4.1_firebase-admin@11.9.0/node_modules/firebase-functions/lib/v2/providers/https.d.ts
    at pp$4.raise (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:21396:13)
    at pp$9.unexpected (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:18607:8)
    at pp$9.expect (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:18601:26)
    at pp$8.parseExportSpecifiers (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:19655:8)
    at Parser.parseExport (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:25858:32)
    at pp$8.parseStatement (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:18783:74)
    at pp$8.parseTopLevel (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:18664:21)
    at Parser.parse (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:18436:15)
    at Function.parse (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:18486:35)
    at Graph.contextParse (file:///Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/rollup@3.25.3/node_modules/rollup/dist/es/shared/node-entry.js:25522:38) {
  pos: 587,
  loc: {
    column: 7,
    file: '/Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/firebase-functions@4.4.1_firebase-admin@11.9.0/node_modules/firebase-functions/lib/v2/providers/https.d.ts',
    line: 13
  },
  raisedAt: 596,
  frame: '11:  * Options that can be set on an onRequest HTTPS function.\n' +
    '12:  */\n' +
    '13: export interface HttpsOptions extends Omit<GlobalOptions, "region"> {\n' +
    '           ^\n' +
    '14:     /**\n' +
    '15:      * If true, do not deploy or emulate this function.',
  id: '/Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/firebase-functions@4.4.1_firebase-admin@11.9.0/node_modules/firebase-functions/lib/v2/providers/https.d.ts',
  hook: 'resolveId',
  code: 'PLUGIN_ERROR',
  plugin: 'commonjs--resolver',
  watchFiles: [
    '/Users/JCGeek/Documents/Projects/nitro/src/index.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/config.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/cli/cli.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/utils/tree.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/types/utils.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/types/fetch.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/preset.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/aws-lambda.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/azure-functions.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/azure.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/base-worker.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/bun.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/cloudflare-module.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/cloudflare-pages.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/cloudflare.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/deno-deploy.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/import-meta.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/deno-server.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/digital-ocean.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/firebase.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/firebase-v2.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/heroku.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/edgio.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/netlify.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/nitro-dev.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/nitro-prerender.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/node-cli.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/node-server.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/node.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/render-com.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/service-worker.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/stormkit.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/types/presets.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/vercel.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/cleavr.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/layer0.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/flightcontrol.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/lagon.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/static.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/github-pages.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/presets/index.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/utils/index.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/externals.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/esbuild.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/virtual/app-config.d.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/config.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/virtual/storage.d.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/storage.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/cache.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/route-rules.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/plugin.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/virtual/plugins.d.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/virtual/error-handler.d.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/virtual/server-handlers.d.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/app.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/renderer.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/types.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/types/handler.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/types/nitro.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/types/global.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/types/h3.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/types/serialize.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/types/index.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/dirs.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/replace.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/virtual.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/dynamic-require.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/externals-legacy.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/timing.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/public-assets.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/server-assets.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/handlers.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/raw.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/storage.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/plugins/app-config.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/rollup/config.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/scan.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/storage.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/compress.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/build.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/imports.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/options.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/nitro.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/dev/vfs.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/dev/error.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/dev/server.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/prerender.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/virtual/public-assets.d.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/runtime/virtual/server-assets.d.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/cli/common.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/cli/commands/dev.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/cli/commands/build.ts',
    '/Users/JCGeek/Documents/Projects/nitro/src/cli/commands/prepare.ts',
    '/Users/JCGeek/Documents/Projects/nitro/node_modules/.pnpm/firebase-functions@4.4.1_firebase-admin@11.9.0/node_modules/firebase-functions/lib/v2/providers/https.d.ts'
  ]
}
```

Reply:
Oh. I think you can leave it as a regular dependency then !